spec/system/community/new_topic_spec.rb

```diff
@@ -62,4 +62,28 @@
       expect(new_topic.topic_category).to eq(category_2)
     end
   end
+
+  context 'when personal coaches are assigned' do
+    let(:coach_1) { create :faculty, school: school }
+    let(:coach_2) { create :faculty, school: school }
+
+    before do
+      create :faculty_course_enrollment, faculty: coach_1, course: course
+      create :faculty_startup_enrollment, faculty: coach_2, startup: student.startup
+    end
+
+    scenario 'when user creates a new topic' do
```

Comment:
@bodhish A scenario is not a context. This should probably have been something along the lines of:

```ruby
scenario 'user creates a new topic that automatically subscribes personal coaches' do
```